blacktie/tx/src/main/cpp/HttpTxManager.cxx

```diff
@@ -379,6 +379,7 @@ bool HttpTxManager::handle_request(
 					LOG4CXX_DEBUG(httptxlogger, "Return 200 OK with body: " <<
 						status << " XA status: " << res);
 					code = 200;
+					_branches.erase(branch->get_name());
```

Comment:
this change is wrong as the branch looks like it has been delete after calling the do_commit or do_rollback. I have a fix in https://github.com/jbosstm/narayana/pull/828